macros/LearnSidebar.ejs

```diff
@@ -1724,6 +1780,20 @@ var text = mdn.localStringMap({
         </ol>
     </details>
   </li>
+  <li><a href="<%=baseURL%>Tools_and_testing/GitHub"><%=text['Git_and_GitHub']%></a></li>
```

Comment:
It works for me, too, but the "Git and GitHub" link is now the only one without sub pages and thus it looks a bit lost to me.

![learnsidebar](https://user-images.githubusercontent.com/349114/79743740-658c7380-8305-11ea-8ec0-e7e339dc26ee.png)

Reply:
Yeah, I agree with you, but I wasn't sure what to do about that. I mean, it is a page at the module level, but it doesn't link to articles on our site — it links to external GitHub articles.

Maybe I could just remove it.

Reply:
Or you add the guides listed here? http://wiki.localhost.org:8000/en-US/docs/Learn/Tools_and_testing/GitHub#Guides

Reply:
OK, I've just tried adding them. I've not added in language strings for the external articles, as they are not translated. I'll try testing the update now.

Reply:
Dammnit, I can't seem to test it. I pulled the changes to my local kumascript submodule, then restarted docker, but it won't show me the updates — it just shows me the original sidebar update as it was at the start of this PR....